app/build.gradle.kts

```diff
@@ -174,6 +174,9 @@ dependencies {
     implementation(libs.googleid)
     implementation(libs.accompanist.permissions)
     implementation(libs.play.services.location)
+    implementation(libs.gson)
+
+
```

Comment:
Question: Is there a reason to use `gson` and not `android.util.JsonReader` (overpass API) or `kotlinx.serialization` (elevation API) ?
Nitpick: you added spaces here.

Reply:
Thanks for the explanation! I think it's fine to have it. Just wanted to know why.